changelog.txt

```diff
@@ -1,3 +1,12 @@
+2.7.0
+=============
+- **Debugging into MSAL was not possible**. MSAL now supports source link. This means developers can debug into MSAL, using VS 15.7+, which will allow them to generate a snukpg package that contains the pdb files (one per platform) and they can add a manifest containing urls to source files in the nuget package. Use the NuGet package explorer to see this manifest. [MSAL PR](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/750)
```

Comment:
this is fantastic, @jennyf19.

I would just replace "Debugging into MSAL was not possible" by "You can now step into MSAL.NET's source code from your application consuming it with NuGet" or something like that

I would replace **MSAL did not support Xamarin.Mac** by **MSAL.NET not supports Xamarin.Mac**

And **Migration from ADALv2 to MSALv2 was not possible due to exposure of the refresh token in ADALv2** by **Easier migration from ADALv2 to MSALv2 due to a new AcquireTokenFromRefreshToken API**

the others are perfect.

The reason why I propose to turn these 3 as positive sentences is they are new features. For the bugs it's ok to keep what you had.

Thanks very much!

Reply:
Great feedback...have updated the file.